projects/picker/src/lib/date-time/date-time-inline.component.ts

```diff
@@ -199,6 +199,17 @@ export class OwlDateTimeInlineComponent<T> extends OwlDateTime<T> implements OnI
     }
   }
 
+  /**
+   * Limit to the amount of days that can be selected at once.
+   * */
```

Comment:
```suggestion
   */
```

Reply:
```suggestion
  /**
   * Limit to the amount of days that can be selected at once.
   * */
  @Input()
  public rangeLimit: number | null = null;

  /**
   * Variable to hold the old max date time value for when we override it with rangeLimit
   * */
  private _oldMaxDateTime: T | null;

  /**
   * Emits selected year in multi-year view
   * This doesn't imply a change on the selected date.
   * */
  @Output()
  yearSelected = new EventEmitter<T>();
```
All the other ones have 2 so maybe I should just add to the one on line 210?

Reply:
Yuck 🙈 
I'll note that down and fix it soon. Leave it as is then.